### ISSUE TITLE
Logitech Logitech Dual Action gamepad fails

### ISSUE THREAD

**Issue by jcoffland** [buildbotics/bbctrl-firmware#52]
This device loads and then the kernel immediately disconnects it.  Not sure why.  Might need to upgrade to a newer distro to sovle this.

```
Aug 20 23:51:59 bbctrl kernel: [   55.819979] usb 1-1.4: USB disconnect, device number 4
Aug 20 23:52:01 bbctrl kernel: [   57.848929] usb 1-1.4: new full-speed USB device number 5 using dwc_otg
Aug 20 23:52:01 bbctrl kernel: [   57.963176] usb 1-1.4: New USB device found, idVendor=046d, idProduct=c216
Aug 20 23:52:01 bbctrl kernel: [   57.963197] usb 1-1.4: New USB device strings: Mfr=1, Product=9, SerialNumber=3
Aug 20 23:52:01 bbctrl kernel: [   57.963210] usb 1-1.4: Product: Logitech Dual Action
Aug 20 23:52:01 bbctrl kernel: [   57.963222] usb 1-1.4: Manufacturer: Logitech
Aug 20 23:52:01 bbctrl kernel: [   57.963234] usb 1-1.4: SerialNumber: EAB2A012
Aug 20 23:52:01 bbctrl kernel: [   57.974230] input: Logitech Logitech Dual Action as /devices/platform/soc/3f980000.usb/usb1/1-1/1-1.4/1-1.4:1.0/0003:046D:C216.0002/input/input1
Aug 20 23:52:01 bbctrl kernel: [   57.974850] hid-generic 0003:046D:C216.0002: input,hidraw0: USB HID v1.11 Joystick [Logitech Logitech Dual Action] on usb-3f980000.usb-1.4/input0
Aug 20 23:52:01 bbctrl kernel: [   57.975411] usb 1-1.4: USB disconnect, device number 5
```

**Comment by DougCoffland** [comment]
I have tried several Logitech F310 game pads and they all seem to work fine. Might this be a hardware problem?

**Comment by jcoffland** [comment]
This was fixed when we upgraded the Raspbian distribution on the RPi.  We are now running Linux kernel 4.9.59.  As I noted above the bug was patched in the kernel at 4.2.8.
